02.calendar/cal.rb

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env ruby
+
+# Dateクラスを使うために宣言
+require 'date'
+
+# optparserクラスを使うために宣言
+require 'optparse'
+
+options = {}
+opt = OptionParser.new
+
+opt.on('-m MONTH'){|m|
+    options[:month] = m.to_i
+}
+
+opt.on('-y YEAR'){|y|
+    options[:year] = y.to_i
+}
+
+opt.parse!(ARGV)
+
+if options[:month] == nil
+    options[:month] = Date.today.month
+end
```

Comment:
これでもよいのですが、こういう１行の簡単な条件文であれば後置ifを使うとみやすくなります。

```suggestion
options[:month] = Date.today.month if options[:month].nil?
```